cf_xarray/__init__.py

```diff
@@ -1,2 +1,10 @@
+from pkg_resources import DistributionNotFound, get_distribution
+
 from .accessor import CFAccessor  # noqa
 from .helpers import bounds_to_vertices, vertices_to_bounds  # noqa
+
+try:
+    __version__ = get_distribution("cf_xarray").version
+except DistributionNotFound:
+    # package is not installed
+    pass
```

Comment:
We could be more explicit here 

```suggestion
    __version__ = 'unknown'
```